src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisParametersService.java

```diff
@@ -98,14 +106,18 @@ public static SecurityAnalysisParameters.IncreasedViolationsParameters getIncrea
         return increasedViolationsParameters;
     }
 
-    public static SecurityAnalysisParametersValues getDefaultSecurityAnalysisParametersValues(String provider) {
+    public static SecurityAnalysisParametersValues getDefaultSecurityAnalysisParametersValues(String provider, LimitReductionConfig limitReductionConfig) {
```

Comment:
I agree and did not understand why at first, it looks like it's because we want to use this method in tests: https://github.com/gridsuite/security-analysis-server/blob/f59cd25cf07831bee8d7a0e777311023a72001c1/src/test/java/org/gridsuite/securityanalysis/server/SecurityAnalysisParametersControllerTest.java#L65